### PR TITLE
Add dummy filter and base class for FL

### LINKED ISSUE #4889: Add FL filter class
**Is your feature request related to a problem? Please describe.**
MonaiAlgo includes optional support for filters. 

**Describe the solution you'd like**
Add some example DummyFilter and abstract base class.

**Describe alternatives you've considered**
n/a

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by holgerroth**
Signed-off-by: Holger Roth <hroth@nvidia.com>

Fixes #4889.

### Description

- Add dummy filter and base class for FL.
- Class is being tested in test_fl_monai_algo.py
- Fixes filter config parsing bug in monai_algo.py

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: monai/fl/utils/dummy_filter.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from monai.fl.utils.exchange_object import ExchangeObject
+from monai.fl.utils.filter import Filter
+
+
+class DummyFilter(Filter):
```
Comment (wyli) [inline monai/fl/utils/dummy_filter.py:5]: I'd suggest the class name `IdentityFilter` or `PrintingFilter` to give some information about what the class do

File: monai/fl/utils/filter.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import abc
+
+from monai.fl.utils.exchange_object import ExchangeObject
+
+
+class Filter(abc.ABC):
```
[the remark this answers is not in the document]
Comment (holgerroth) [reply monai/fl/utils/filter.py]: Good idea. I moved both to filters.py

File: monai/fl/utils/dummy_filter.py (hunk 1 above)
Comment (holgerroth) [reply monai/fl/utils/dummy_filter.py]: Good point. I went for `SummaryFilter`.

**Review by Nic-Ma — APPROVED** [review]

**Pull request merged**